x-pack/plugins/cases/public/components/configure_cases/index.tsx

```diff
@@ -222,24 +222,25 @@ export const ConfigureCases: React.FC = React.memo(() => {
                 </EuiCallOut>
               </SectionWrapper>
             )}
+            {/* should these include push? */}
```

Comment:
@cnasikas Same question as above about the configuration page of cases, should we require `push` permissions? I'm leaning towards no.